pkg/configgen/configgen.go

```diff
@@ -130,6 +134,8 @@ func (g *ConfigGenerator) Daemonset() *appsv1.DaemonSet {
 			},
 		})
 	}
+	projectedConfigVolumes = g.injectCertVolumes(projectedConfigVolumes)
+	projectedConfigVolumes = g.injectRadiusClientVolume(projectedConfigVolumes)
```

Comment:
I found this a bit confusing - Can we rename these functions to 'append*Volume[s]`